hawkular-alerts-engine/pom.xml

```diff
@@ -227,12 +227,14 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
+              <!--
               <systemProperties>
                 <property>
                   <name>testOutput</name>
                   <value>${project.build.testOutputDirectory}</value>
                 </property>
               </systemProperties>
+              -->
```

Comment:
+1
Thanks, this was leave by me while debugging an strange error on the rest profile, I should clean it.